include/openssl/ssl.h

```diff
@@ -3593,6 +3593,14 @@ OPENSSL_EXPORT void SSL_set_psk_client_callback(
                              unsigned max_identity_len, uint8_t *psk,
                              unsigned max_psk_len));
 
+// SSL_psk_client_cb_func defines a function signature for the client callback.
```

Comment:
```suggestion
// SSL_psk_client_cb_func defines a function signature for |SSL(_CTX)_psk_client_cb_func|
```

Should we change `SSL(_CTX)_psk_client_cb_func` to just use this function signature? Same comments with `SSL_psk_server_cb_func` as well.

Reply:
yep, that makes sense. updated function signatures for relevant functions.